### ISSUE TITLE
[Bug] [UWP] SfListView is not interactable when Xamarin.Forms is upgraded to 4.5.0.356

### ISSUE THREAD

**Issue by Chansardrake** [xamarin/Xamarin.Forms#9829]
### Description
SfListView items do have touch/click interactions on target versions

- Windows 10, Fall Creators Update (10.0; Build 16299)
- Windows 10, version 1803 (10.0; Build 17134)

We face the same issue in SfAccordion also.

### Steps to Reproduce

1. Run the sample
2. Click the item in SfListView

### Expected Behavior
SfListView item is selected
### Actual Behavior
SfListView item is not selected
### Basic Information

- Version with issue: 4.5.0.356
- Last known good version: 4.4.0.991757
- IDE: Visual Studio 2017

Sample Link: [Sample](https://www.syncfusion.com/downloads/support/directtrac/general/ze/UWP_Bug856398768.zip) 

### Screenshots
![image](https://user-images.githubusercontent.com/21023069/75847489-5f017580-5e05-11ea-9f72-3933d388777a.png)

### Workaround

Issue is resolved when changing the target version to 

- Windows 10, version 1809 (10.0; Build 17763)
- Windows 10, version 1903 (10.0; Build 18362)

**Comment by hartez** [comment]
I don't think this has anything to do with the SFListView - I think the issue is that the SyncFusion license acceptance dialog does not show up when the project is started using 16299 and 17134.

**Comment by hartez** [comment]
If I downgrade to 4.4 and target 16299, the license acceptance dialog shows up again; after clicking Ok the controls are interactive. So something in 4.5 is preventing the dialog from appearing.

**Comment by hartez** [comment]
Can't tell for sure, but it looks like SF might be using a translucent modal to prevent the controls from being used without accepting the license. (Start the repro app in a small window and then expand it - the overlay won't expand with the rest of the window, and you'll be able to interact with the controls outside of it.)

![uwp_sf](https://user-images.githubusercontent.com/538025/75896945-0629e100-5df5-11ea-96a4-07aa4b4d8e82.png)

**Comment by bmacombe** [comment]
@hartez Probably #9606
